app-arch/bzip2/bzip2-1.0.8-r1.ebuild

```diff
@@ -41,6 +43,28 @@ src_prepare() {
 		-e 's:ln -s -f $(PREFIX)/bin/:ln -s -f :' \
 		-e 's:$(PREFIX)/lib:$(PREFIX)/$(LIBDIR):g' \
 		Makefile || die
+
+	if use prefix ; then
+		hprefixify -w "/^PATH=/" bz{diff,grep,more}
+
+		# This is a makefile for Darwin, which already "includes" sane so
+		cp "${FILESDIR}"/${PN}-1.0.6-Makefile-libbz2_dylib Makefile-libbz2_dylib || die
+
+		if [[ ${CHOST} == *-hpux* ]] ; then
+			sed -i -e 's,-soname,+h,' Makefile-libbz2_so || die "cannot replace -soname with +h"
+			if [[ ${CHOST} == hppa*-hpux* && ${CHOST} != hppa64*-hpux* ]] ; then
+				sed -i -e '/^SOEXT/s,so,sl,' Makefile-libbz2_so || die "cannot replace so with sl"
+				sed -i -e '/^SONAME/s,=,=${EPREFIX}/lib/,' Makefile-libbz2_so || die "cannt set soname"
+			fi
+		fi
+
+		if [[ ${CHOST} == *-cygwin* ]] ; then
```

Comment:
this is already conditionalised (for a Prefix-only target) so no need for conditional

-> drop the if use prefix